hw/top_earlgrey/cdc/cdc_waivers.w_net_no_wave.tcl

```diff
@@ -0,0 +1,8 @@
+# Copyright lowRISC contributors.
+# Licensed under the Apache License, Version 2.0, see LICENSE for details.
+# SPDX-License-Identifier: Apache-2.0
+#
+# Verix CDC waiver file
+
+set_rule_status -rule {S_NET_NO_WAVE} -expression {(Signal =~ "ast_init_done[3:1]") && (DriverType =~ "Undriven") && (FanoutClockDomain == "IO_DIV4_CLK")} -status {Deferred} -comment {Deferred to Post-M2 while waiting for Nuvoton to clarify}
```

Comment:
Can you file an M3 issue for this and mention the issue number here?

Reply:
Yes, I created #16478 and added it in this comment